src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.subway.path.ui;
+
+import nextstep.subway.path.application.PathService;
+import nextstep.subway.path.dto.PathResponse;
+import nextstep.subway.path.exception.SourceEqualsWithTargetException;
+import nextstep.subway.path.exception.StationNotExistsException;
+import nextstep.subway.path.exception.StationsNotConnectedException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PathController {
+    private PathService pathService;
+
+    public PathController(PathService pathService) {
+        this.pathService = pathService;
+    }
+
+    @RequestMapping("/paths")
+    public ResponseEntity getShortestPath(@RequestParam Long sourceId, @RequestParam Long targetId) {
```

Comment:
```suggestion
    public ResponseEntity<PathResponse> getShortestPath(@RequestParam Long sourceId, @RequestParam Long targetId) {
```

요기서도 제너릭 명시화 해주시면 좋을 것 같아요! 참고로 리턴값이 없을 때는 `ResponseEntity<Void>` 처럼 작성하실 수도 있답니다 😃